tests/golos.publication_rewards_tests.cpp

```diff
@@ -1194,9 +1201,61 @@ BOOST_FIXTURE_TEST_CASE(golos_delegators_test, reward_calcs_tester) try {
     delegator3_amount = vest.get_balance_raw(_users[10])["vesting"].as<asset>();
     produce_blocks(golos::seconds_to_blocks(post.window));
     BOOST_CHECK_GT(vest.get_balance_raw(_users[1])["vesting"].as<asset>(), voter_amount);
-    BOOST_CHECK_GT(vest.get_balance_raw(_users[8])["vesting"].as<asset>(), delegator_amount);
-    BOOST_CHECK_GT(vest.get_balance_raw(_users[9])["vesting"].as<asset>(), delegator_amount);
-    BOOST_CHECK_EQUAL(vest.get_balance_raw(_users[10])["vesting"].as<asset>(), delegator_amount);
+    BOOST_CHECK_GT(vest.get_balance_raw(_users[8])["vesting"].as<asset>(), delegator1_amount);
+    BOOST_CHECK_GT(vest.get_balance_raw(_users[9])["vesting"].as<asset>(), delegator2_amount);
+    BOOST_CHECK_EQUAL(vest.get_balance_raw(_users[10])["vesting"].as<asset>(), delegator3_amount);
+    check();
+
+    BOOST_TEST_MESSAGE("--- add_funds_to_forum");
+    BOOST_CHECK_EQUAL(success(), add_funds_to_forum(50000));
+    check();
+    
+    BOOST_TEST_MESSAGE("--- checking that interest_rate < 1");
+    BOOST_TEST_MESSAGE("--- create_message: " << name{_users[0]}.to_string());
+    BOOST_CHECK_EQUAL(success(), create_message({_users[0], "permlink5"}));
+    check();
+    BOOST_CHECK_EQUAL(success(), delegate_vest(_users[11], _users[1], amount, interest_rate_1prcnt));
+    BOOST_CHECK_EQUAL(success(), delegate_vest(_users[12], _users[1], amount, interest_rate_1prcnt));
+    BOOST_CHECK_EQUAL(success(), delegate_vest(_users[13], _users[1], amount, interest_rate));
```

Comment:
why we expect this delegation values result in 0 interest rate of vote.delegators.user[11] ?

it's impossible to know that values are right without calculation, so we need some documentation, like:
>each user[x] have same vesting = 500.000, and delegators delegate `amount` = 5.000,
so voter's effective vesting = 500+5+5+5 = 515.
let calculate interest rates of delegators in vote:
user11 = user12 = (1% * 5 / 515) = 0.0097%. it less than 0.01% so rounded down to 0.
user13 = (30% * 5 / 515) = … > 0.01%

please add comment about calculation (not sure mine is correct)